testing/file_test/README.md

```diff
@@ -195,6 +195,18 @@ Supported comment markers are:
     `EXTRA-ARGS` can be specified at most once, and a test cannot specify both
     `ARGS` and `EXTRA-ARGS`.
 
+-   ```
+    // INCLUDE-FILE: <path/from/repository/root>
+    ```
+
+    Includes the specified file in the test's virtual file system and adds the
+    path to the test's arguments.
+
+    This can be used to provide a minimal `Core` package (and `prelude` library)
+    in toolchain tests by providing a path to a file with
+    `package Core library "prelude";` as its first line and adding
+    `--custom-core` to `EXTRA-ARGS`.
```

Comment:
This README is intended to be for the common infrastructure, whereas this is a way the toolchain in specific uses it. Maybe the note should be moved to toolchain documentation, e.g. https://github.com/carbon-language/carbon-lang/blob/trunk/toolchain/docs/adding_features.md#tests-and-debugging ?

Reply:
Right, I would like to keep something here just to help people discover it. But we should also document it toolchain/docs/, will do, thanks.

Reply:
Had you considered a note saying like "See toolchain-specific features" (linked?)

Now it feels a bit like there's divergent documentation, since you have different information here.

Reply:
That's fair: https://github.com/carbon-language/carbon-lang/pull/5105